obspy/io/gse2/libgse2.py

```diff
@@ -465,13 +465,33 @@ def parse_sta2(line):
     """
     header = {}
     try:
+        lat = line[15:24].strip()
+        if lat:
+            lat = float(lat)
+        else:
+            lat = None
+        lon = line[25:35].strip()
+        if lon:
+            lon = float(lon)
+        else:
+            lon = None
+        lon = lon and float(lon)
+        elev_edepth = [x.strip() for x in line[48:].strip().split()]
```

Comment:
No need to `.strip()` if you're going to `.split()` since it compresses whitespace anyway.